### PR TITLE
Add helm chart for vineyard operator

### PR DESCRIPTION
**Pull request by dashanji**
As titiled.

### CODE REVIEW

File: charts/vineyard-operator/Chart.yaml [hunk 1]
```diff
@@ -30,10 +24,9 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.8.2
+version: 0.1.0
```
Comment (sighingnow) [inline charts/vineyard-operator/Chart.yaml:27]: I suggest follow the version of vineyard, as we are in the same repo and use the same release cycle.

File: charts/vineyard-operator/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Uninstall
+
+The installed charts can be removed with
+
+```console
+helm uninstall vineyard-operator
+```
+
+More information about the helm chart could be found at [artifacthub][1] and [parameters][2].
```
Comment (sighingnow) [inline charts/vineyard-operator/README.md:24]: Add a section about how to create a vineyard cluster after the vineyard operator ready here.

File: docs/notes/deploy-kubernetes.rst [hunk 3]
```diff
@@ -23,6 +23,18 @@ from containers in the same pod, or pods on the same hosts.
 Deploying with Helm
 ^^^^^^^^^^^^^^^^^^^
 
+Before installing the vineyard operator, you should install creat-manager at first.
```
Comment (sighingnow) [inline docs/notes/deploy-kubernetes.rst:26]: `creat-manager` -> `cert-manager`

File: docs/notes/deploy-kubernetes.rst [hunk 4]
```diff
@@ -31,9 +43,11 @@ with ``helm``:
     helm repo add vineyard https://vineyard.oss-ap-southeast-1.aliyuncs.com/charts/
     helm install vineyard vineyard/vineyard
```
Comment (sighingnow) [inline docs/notes/deploy-kubernetes.rst:44]: I think it should be `helm install vineyard-operator vineyard/vineyard-operator`, as you have removed the existing helm charts.

**Review by sighingnow — CHANGES_REQUESTED** [review]

**Comment by sighingnow** [comment]
I have leaf some comments.

File: docs/notes/deploy-kubernetes.rst (hunk 4 above)
Comment (sighingnow) [reply docs/notes/deploy-kubernetes.rst:44]: And you need to double check the content in `README.rst` as well.

File: charts/vineyard-operator/Chart.yaml (hunk 1 above)
Comment (dashanji) [reply charts/vineyard-operator/Chart.yaml:27]: Get it.

File: charts/vineyard-operator/README.md (hunk 2 above)
Comment (dashanji) [reply charts/vineyard-operator/README.md:24]: Copy.

**Comment by dashanji** [comment]
Comments down, and Fixed the [dependency warning](https://github.com/v6d-io/v6d/security/dependabot/17).

**Review by sighingnow — APPROVED** [review]

**Pull request merged**